### DISCUSSION TITLE
Laravel Livewire Breeze Installation

### DISCUSSION THREAD

**Discussion by tomshaw** [livewire/livewire#7695] [1. Help] [answered]
I just created a new Laravel project using composer `composer create-project laravel/laravel projectname` then installed Laravel Breeze `composer require laravel/breeze --dev` then finally ran `php artisan breeze:install` at which point I choose to install **livewire**. My question is it's not clear if I need to add the Livewire tags @livewireStyles @livewireScripts to my layout.

**Comment by amjadiqbal** [comment, marked as answer]
you don't need to do livewire tags it will be automatically added to your Laravel project. Installation create it's own file into your projects. you can run the project using npm run dev and you will be able to access your project at this port http://localhost:5173/

**Reply by tomshaw** [reply]
Yea I reviewed the source. Was considering the same functionality for tomshaw/mediable. I just find any mention of this in the documentation.
